app/com/arpnetworking/metrics/portal/alerts/impl/CassandraAlertRepository.java

```diff
@@ -109,10 +111,10 @@ public AlertQuery createQuery(final Organization organization) {
         final Mapper<models.cassandra.Alert> mapper = _mappingManager.mapper(models.cassandra.Alert.class);
         final models.cassandra.Alert.AlertQueries accessor = mapper.getManager().createAccessor(models.cassandra.Alert.AlertQueries.class);
         final Result<models.cassandra.Alert> result = accessor.getAlertsForOrganization(query.getOrganization().getId());
-        final List<models.cassandra.Alert> allAlerts = result.all();
+        final Spliterator<models.cassandra.Alert> allAlerts = result.spliterator();
         final int start = query.getOffset().orElse(0);
 
-        Stream<models.cassandra.Alert> alertStream = allAlerts.stream();
+        Stream<models.cassandra.Alert> alertStream = StreamSupport.stream(allAlerts, false);
```

Comment:
This actually performs the filtering operations in Cassandra? -- in particular the `alerts.stream().skip(start)`? -- for future reference can you link a doc here to the Cassandra query language.